Docs/developer_guide/python_faq.md

```diff
@@ -2,6 +2,85 @@
 
 Frequently asked questions about how to write Python scripts for Slicer.
 
+## What is the Slicer Python Environment?
+
+You can consider each Slicer installation as a virtual Python environment - the same way
+as you create virtual environments using python or conda.
+
+## What is the PythonSlicer executable?
+
+`PythonSlicer` is an executable provided in the `bin` directory of the Slicer installation.
+It’s the same version of Python available through the Slicer application, but it acts like
+a normal `python` (or `python3`) executable in terms of running scripts and can be used t
+ access all the packages installed in Slicer.
+
+This means you can use it in place of regular python to make use of the Slicer environment
+for batch processing and command-line operations.
+
+:::{warning}
+The application context is **not** available. This means that many objects like the running
+application, the MRML scene and the loaded modules are not available.
+:::
+
+:::{tip}
+You may use {func}`slicer.util.pip_install()` to install additional packages.
+:::
+
+## What is the Python Console?
+
+This describes the [](/user_guide/user_interface.md#python-console) available through
```

Comment:
Is the empty `[]` valid syntax?

Reply:
It is indeed, it means that corresponding header is used in the generated text. 

> If no text is given or it is an auto-link, e.g. `[](#dest)` or `<project:#dest>`, then MyST will attempt to resolve an implicit text. For example, if the destination is a heading, then the heading text will be used as the link text, or if the destination is a figure/table then the caption will be used as the link text. Otherwise, the link text will be the destination itself.

See https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#explicit-vs-implicit-link-text

Reply:
Thanks for the explanation.